### ISSUE TITLE
Re-implement cover mode (folder swipe-up gestures)

### ISSUE THREAD

**Issue by Noklle** [Deletescape-Media/Lawnchair#2659]
Just switched over from the Play Store version of Launchair to the latest alpha over on Telegram and I'm kinda dissapointed to see that cover mode seems to be gone, unlesss I just can't find the option somehow. It's a neat feature that I currently use in the older version of Lawnchair and I'd love to see it return

**Comment by RikKoedoot** [comment]
Lawnchair has been rewritten since v11, this is why it's not there yet. It's the idea to add the old features back, so it will come back in the future

**Comment by 4-FLOSS-Free-Libre-Open-Source-Software** [comment]
- #933
- #1479
- #2128

**Comment by Amorphous404** [comment]
It seems i added this too with my other folder related Features, so just adding the reference #2843 (Point No 4), Or Should I Remove It?

**Comment by Unkn0wnCat** [comment]
bump

**Comment by 4-FLOSS-Free-Libre-Open-Source-Software** [comment]
> bump

**Comment by VictorVow** [comment]
Bump

**Comment by 4-FLOSS-Free-Libre-Open-Source-Software** [comment]
Remove stale label

**Comment by mksy** [comment]
are the future has come?

**Comment by vital987** [comment]
Any progress?

**Comment by RoL0NL** [comment]
Progress? Now that 14 has released..

**Comment by Jamminroot** [comment]
Some time into the future passed, any luck?

**Comment by non-bin** [comment]
Bump

**Comment by jvdmeij** [comment]
Would be great to get this feature on the roadmap. Makes the launcher so much cleaner!

**Comment by 4-FLOSS-Free-Libre-Open-Source-Software** [comment]
Absolutely, I am in favor of bringing back cover mode to restore a feature that was much appreciated.

Well, version 9.1 alpha 3 still has this working feature.

**Comment by jvdmeij** [comment]
> Absolutely, I am in favor of bringing back cover mode to restore a feature that was much appreciated.
> 
> Well, version 9.1 alpha 3 still has this working feature.

Currently running the playstore version. Is there something newer?

**Comment by cjseaslug** [comment]
I would love to see this feature implemented! Would this require a lot of expansion or reworking of icons/gesture logic?

**Comment by sharko789** [comment]
I would love to see this implemented as well, it's the only thing keeping me on the legacy version. Also the discussion in #4651 seems to have shifted to the same topic.

**Comment by GhandiSenpai** [comment]
I'd also love to see this feature implemented, it's the only feature that I'm missing to finally move away from action launcher.

**Comment by richlux** [comment]
Please, please re-implement this. I'm stuck on the legacy version until this is supported.

Rich

**Comment by brokoli-mate** [comment]
Please reimplement this

**Comment by 4-FLOSS-Free-Libre-Open-Source-Software** [comment]
I'm stuck on the legacy version 9.1 alpha 3 until this is supported.
- #933
- #1479
- #2128
- #4651 
- #6433 
- #6704

**Comment by 4-FLOSS-Free-Libre-Open-Source-Software** [comment]
I would also like to see Cover Mode / folder swipe-up gestures reimplemented, as this is still the main reason I cannot upgrade from the older Lawnchair version. As far as I can tell, none of the current Lawnchair-15-Beta-3-Release-Notes releases include the old behavior where tapping a folder opens the first app directly while swiping opens the folder. Without this feature, updating would be a functional downgrade for users who rely on Cover Mode as part of their daily workflow.

**Comment by sharko789** [comment]
It seems like there is finally some work being done related to this in #6964 and #6958 or am I mistaken?
Any comments from @validcube or @SuperDragonXD?
